packages/clarity-js/src/data/metric.ts

```diff
@@ -5,7 +5,9 @@ import encode from "./encode";
 export let data: MetricData = null;
 export let updates: MetricData = null;
 let metricMap: WeakMap<Node, string> = null; // Maps metric nodes => innerText
-const regex = /[^0-9\.]/g;
+const numberRegex = /[^0-9\.,]/g;
+const digitsRegex = /[^0-9\.]/g;
```

Comment:
What's the diff between a number regex and a digits regex in semantics? I see the difference in regex, but not clear what you intend from names.

Reply:
I will rename the variables to be more clear. In one case we remove commas (for en fallback if Intl.NumberFormat is not present) and in another we keep the comma to handle it properly respecting page language.

Reply:
Done.